bundles/org.openhab.binding.amberelectric/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,96 @@
+			<channel id="controlled-load-status" typeId="controlled-load-status"/>
+			<channel id="feed-in-price" typeId="feed-in-price"/>
+			<channel id="feed-in-status" typeId="feed-in-status"/>
+			<channel id="nem-time" typeId="nemtime"/>
+			<channel id="spike" typeId="spike"/>
+			<channel id="renewables" typeId="renewables"/>
+		</channels>
+
+		<config-description>
+			<parameter name="apiKey" type="text" required="true">
+				<label>API Key</label>
+				<description>API key from the Amber website</description>
+			</parameter>
+			<parameter name="nmi" type="text">
+				<label>NMI</label>
+				<description>NMI for your address (Optional)</description>
+			</parameter>
+			<parameter name="refresh" type="integer" min="60" unit="s">
+				<label>Refresh Interval</label>
+				<description>Specifies the refresh interval in seconds.</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="electricity-price">
+		<item-type>Number:EnergyPrice</item-type>
+		<label>Current Electricity Price</label>
+		<description>Current price to import power from the grid</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="controlled-load-price">
+		<item-type>Number:EnergyPrice</item-type>
+		<label>Current Controlled Load Price</label>
+		<description>Current price to import power for Controlled Load</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="feed-in-price">
+		<item-type>Number:EnergyPrice</item-type>
+		<label>Current Feed-In Price</label>
+		<description>Current price to export power to the grid</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+	<channel-type id="electricity-status">
+		<item-type>String</item-type>
+		<label>Current Electricity Status</label>
+		<description>Current price status of grid import</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+	<channel-type id="controlled-load-status">
+		<item-type>String</item-type>
+		<label>Current Controlled Load Status</label>
+		<description>Current price status of Controlled Load</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+	<channel-type id="feed-in-status">
+		<item-type>String</item-type>
+		<label>Current Feed-In Status</label>
+		<description>Current price status of Feed-In</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+	<channel-type id="nem-time">
+		<item-type>String</item-type>
+		<label>NEM Time</label>
+		<description>NEM time of last pricing update</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+	<channel-type id="renewables">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Current Renewables</label>
+		<description>Current level of renewables in the grid</description>
+		<state readOnly="true" pattern="%d %%"/>
```

Comment:
When unit is provided for the item, there is probably no need to enforce display pattern:
```suggestion
		<state readOnly="true" pattern="%d %unit%"/>
```

Reply:
Done

Reply:
Looks not pushed?

Reply:
Really done now :) No idea what happened there